### PR TITLE
fire inc and sum metrics

### PR DESCRIPTION
**Pull request by rudigiesler**
Make the existing functions fire both the inc (for totals) and sum (for time frame totals) metrics.

### CODE REVIEW

**Comment by rudigiesler** [comment]
@hodgestar ready for review

File: lib/metrics-helper.js [hunk 1]
```diff
@@ -18,15 +18,17 @@ var MetricsHelper = Eventable.extend(function(self, im) {
...
+            started with a new user.
 
             :param string label:
                 The label for the metric. Defaults to ``unique_users``
         */
         label = typeof label !== 'undefined' ? label: 'unique_users';
         self.im.on('user user:new', function(e) {
-            return e.user.im.metrics.fire.inc(label, 1);
+            return e.user.im.metrics.fire.inc(label, 1).then(function() {
+                return e.user.im.metrics.fire.sum(label + '.transient', 1);
```
Comment (hodgestar) [inline lib/metrics-helper.js]: It might be cool to fire these operations simultaneously and use `Q.all` to wait on the results.

**Comment by hodgestar** [comment]
Changes look good. Left one comment suggesting using `Q.all`.

**Comment by rudigiesler** [comment]
@hodgestar changed to using `Q.all`, ready for review again

**Comment by hodgestar** [comment]
:+1:

**Pull request merged**